Source/UserManagement/Domain/StaffUser/AggregateRoots/StaffUserManagement.cs

```diff
@@ -0,0 +1,161 @@
+                case Role.DataConsumer:
+                    HandleAddDataConsumer(command);
+                    break;
+                case Role.DataCoordinator:
+                    HandleAddDataCoordinator(command);
+                    break;
+                case Role.DataOwner:
+                    HandleAddDataOwner(command);
+                    break;
+                case Role.DataVerifier:
+                    HandleAddDataVerifier(command);
+                    break;
+                case Role.SystemCoordinator:
+                    HandleAddSystemCoordinator(command);
+                    break;
+
+                default:
+                    //TODO: This should not happen due to InputValidation
+                    break;
+                
+            }
+        }
+
+        public void AddPhoneNumber(AddPhoneNumberToStaffUser command)
+        {
+            Apply(new PhoneNumberAddedToStaffUser(
+                command.StaffUserId, command.PhoneNumber, (int)command.Role
+                ));
+        }
+
+        public void RemovePhoneNumber(RemovePhoneNumberFromStaffUser command)
+        {
+            Apply(new PhoneNumberRemovedFromStaffUser(
+                command.StaffUserId, command.PhoneNumber, (int)command.Role
+                ));
+        }
+
+        private void HandleAddAdmin(AddStaffUser command)
+        {
+            Apply(new AdminAdded(
+                command.StaffUserId, command.FullName,
+                command.DisplayName, command.Email
+                ));
+
+        }
+
+        private void HandleAddDataConsumer(AddStaffUser command)
+        {
+            Apply(new DataConsumerAdded(
+                command.StaffUserId, command.FullName,
+                command.DisplayName, command.Email,
+                command.Location.Longitude, command.Location.Latitude
+                ));
+        }
+        private void HandleAddDataCoordinator(AddStaffUser command)
+        {
+            Apply(new DataCoordinatorAdded(
+                command.StaffUserId, command.FullName, command.DisplayName,
+                command.Email, command.YearOfBirth, (int)command.Sex, command.NationalSociety,
+                (int)command.PreferredLanguage, command.Location.Longitude,
+                command.Location.Latitude
+                ));
+
+            if (command.MobilePhoneNumber != null && command.MobilePhoneNumber.Count > 0)
+            {
+                foreach (var number in command.MobilePhoneNumber)
+                {
+                    Apply(new PhoneNumberAddedToStaffUser(
+                        command.StaffUserId, number,
+                        (int)command.Role
+                    ));
+                }
+            }
+            //TODO: DO the same for NationalSocieties
+        }
+        private void HandleAddDataOwner(AddStaffUser command)
+        {
+            Apply(new DataOwnerAdded(
+                command.StaffUserId, command.FullName, command.DisplayName,
```

Comment:
For code readability, you should add each parameter on its own line if you start breaking it up (personal preference, but also related to how the human eye moves when reading)